explore/src/main/scala/explore/itc/package.scala

```diff
@@ -6,20 +6,23 @@ package explore.itc
 import cats.data.NonEmptyChain
 import explore.Icons
 import explore.components.ui.ExploreStyles
+import explore.model.syntax.all.*
 import japgolly.scalajs.react.vdom.html_<^.*
+import lucuma.core.enums.CalibrationRole
 import lucuma.itc.ItcCcd
 import lucuma.react.fa.*
 import lucuma.react.floatingui.syntax.*
 import lucuma.ui.syntax.all.given
 
-// Icon to indicate a field is required to do ITC calculations
-def requiredForITC: TagMod =
-  <.span(
-    LayeredIcon(fixedWidth = true, clazz = ExploreStyles.WarningIcon)(
-      Icons.StarExclamation.withSize(IconSize.X1),
-      TextLayer("ITC", clazz = ExploreStyles.RequiredForItcText, inverse = true)
-    )
-  ).withTooltip("Required for ITC")
+extension (role: Option[CalibrationRole])
+  // Icon to indicate a field is required to do ITC calculations
+  def requiredForITC: TagMod =
```

Comment:
Can we name this with a verb describing what it does? `renderRequiredForITCIcon` or similar?